cms/static/sass/_variables.scss

```diff
@@ -209,12 +210,16 @@ $ui-link-color: $blue-u2;
 $ui-link-color-focus: $blue-s1;
 
 // +Specific UI
-// ==================== 
+// ====================
 $ui-notification-height: ($baseline*10);
 $ui-update-color: $blue-l4;
 
-// +Deprecated 
-// ==================== 
+// +Paths
+// ====================
+$static-path: '..' !default;
```

Comment:
Please move this section of config/variables to the top of this Sass partial (to be consistent with its placement in LMS-centric Sass config).